desk/ted/vitals/connection-check.hoon

```diff
@@ -12,9 +12,9 @@
 ^-  form:m
 =+  !<([~ target=ship] arg)
 ;<  our=@p  bind:m  get-our:io
+;<  now=@da  bind:m  get-time:io
```

Comment:
I don't like the deletions of the `;<  now=@da  bind:m  get-time:io` lines below in favor of this "eager" assignment.

In practice it's fine, because the `now` only gets used as part of the returned value.  
But in theory the early/eager fetching of a "current time" in the context of threads, especially if it's followed by other `;<`s, is suspicious. General expectation is that computation inside of a `;<` may be asynchronous, which mean they may make the `now` value stale, because time has passed.  
Again, in practice accuracy of that value isn't _super_ important here, but I think it's good ~~practice~~ theory to keep the `get-time:io` calls as close to the use of the resulting time as possible.

Reply:
Understood; I'll put this back in order so that `now` time reports are more accurate.